### ISSUE TITLE
How to train high resolution images?

### ISSUE THREAD

**Issue by fanyuanxiang** [PaddlePaddle/PaddleDetection#6610]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

Training high resolution images, such as 1920*1080, to detect target object size is basically the same  size (300*300) , how to adjust the parameters training.

**Comment by zoooo0820** [comment]
Is the problem about GPU memory or model accuracy?
- If it is about GPU memory, just set lower `batch_size` and adjust the `learning rate`.
- If it is about model accuracy, in my opinion, the 300*300 size seems well. But you can still refer [this config](https://github.com/PaddlePaddle/PaddleDetection/blob/develop/configs/visdrone/ppyoloe_crn_l_p2_alpha_largesize_80e_visdrone.yml), which is to deal with the small size object and trained with a large input image size.
